contentcuration/contentcuration/frontend/shared/views/ImmersiveModalLayout.vue

```diff
@@ -0,0 +1,79 @@
+<template>
```

Comment:
Would it make sense to put this under a `views/layouts` folder so they are somewhat separated from the smaller components?

Reply:
yes, or even just under 'layouts'. I didn't want to shuffle too much around in shared just yet